src/raftstore/store/snap.rs

```diff
@@ -704,14 +706,22 @@ impl Snap {
             } else {
                 let mut key_count = 0;
                 let mut size = 0;
+                let base: i64 = 128 * 1024;
```

Comment:
The value of `base` should depending on `refill_bytes_per_period`, it's better make it configurable.

Reply:
Sure, i will fix this later.